ksql-engine/src/main/java/io/confluent/ksql/KsqlEngine.java

```diff
@@ -312,6 +312,66 @@ public QueryMetadata getQueryExecutionPlan(final Query query, final KsqlConfig k
     }
   }
 
+  public List<Pair<String, Statement>> parseStatements(
+      final String queriesString,
+      final MetaStore tempMetaStore
```

Comment:
for the context of this method, is this just `metaStore`, rather than `tempMetaStore`?

Reply:
Yes, it's not the same as the `metastore` in `KsqlEngine` class.